mne/utils/_logging.py

```diff
@@ -124,18 +124,41 @@ def %(name)s(%(signature)s):\n
     return fm.make(body, evaldict, addsource=True, **attrs)
 
 
-class use_log_level(object):
-    """Context handler for logging level.
+class use_log_level:
+    """Context manager for logging level.
 
     Parameters
     ----------
     level : int
```

Comment:
Here it says it must be an `int`; in your new example it's a `boolean`; and in real-life I'd always use a string like `'warning'` or `'error'` … time to consolidate a little?

Reply:
For consistency I just renamed `level -> verbose` and used `%(verbose)s`. This function wasn't really meant to be public before, I really doubt anyone did `use_log_level(level=whatever)` rather than just `use_log_level(whatever)`, so I think it's an okay backward-incompat arg name rename